src/action/wallet.js

```diff
@@ -1,13 +1,66 @@
-import { toBuffer, parseSat, checkHttpStatus } from '../helper';
+import { toBuffer, parseSat, checkHttpStatus, nap } from '../helper';
+import { MIN_PASSWORD_LENGTH, NOTIFICATION_DELAY } from '../config';
 import * as log from './log';
 
 class WalletAction {
-  constructor(store, grpc, notification) {
+  constructor(store, grpc, nav, notification) {
     this._store = store;
     this._grpc = grpc;
+    this._nav = nav;
     this._notification = notification;
   }
 
+  //
+  // Verify Seed actions
+  //
+
+  initSeedVerify() {
+    this._store.wallet.seedVerify = ['', '', ''];
+    this._nav.goSeedVerify();
+  }
+
+  setSeedVerify({ word, index }) {
+    this._store.wallet.seedVerify[index] = word;
+  }
+
+  //
+  // Wallet Password actions
+  //
+
+  initSetPassword() {
+    this._store.wallet.password = '';
+    this._store.wallet.passwordVerify = '';
+    this._nav.goSetPassword();
+  }
+
+  initPassword() {
+    this._store.wallet.password = '';
+    this._nav.goPassword();
+  }
+
+  setPassword({ password }) {
+    this._store.wallet.password = password;
+  }
+
+  setPasswordVerify({ password }) {
+    this._store.wallet.passwordVerify = password;
+  }
+
+  //
+  // Wallet actions
+  //
+
+  async init() {
+    try {
+      await this.generateSeed();
+      this._nav.goLoader();
+      await nap(NOTIFICATION_DELAY);
+      this._nav.goSeed();
+    } catch (err) {
```

Comment:
I ran into an error here: "wallet already exists" -- had to delete my existing `.config/lightning-app` to get to the onboarding.

Should we be logging this error to the console and/or putting a note in the README for people who have previously downloaded the app?

Reply:
I think it should be good. Not too many testers yet.